src/app.js

```diff
@@ -15,6 +28,14 @@ app.use(express.static('public'))
 app.use(favicon(__dirname + '/public/favicon.ico'));
 
 // routes
-app.use('/api/v1', mainRouter);
+app.use('/api/v1/auth', authRouter);
+app.use('/api/v1', authenticateUser, mainRouter);
+app.use('/api/v1/vendors', authenticateUser, vendorRouter);
+app.use('/api/v1/services', authenticateUser, serviceRouter);
```

Comment:
services don't have to be protected. we want to let people browse through the app without authorization

Reply:
It is a stub, need to discuss the general philosophy what the client/vendor should see, have access to etc